lighthouse-treemap/app/src/main.js

```diff
@@ -110,13 +121,40 @@ class TreemapViewer {
     };
   }
 
-  show() {
-    const group = 'scripts';
+  createViewModes() {
+    /** @type {LH.Treemap.ViewMode[]} */
+    const viewModes = [];
 
-    const depthOneNodes = this.depthOneNodesByGroup[group];
-    this.currentTreemapRoot = this.wrapNodesInNewRootNode(depthOneNodes);
-    renderViewModeOptions(this.currentTreemapRoot.resourceBytes);
+    viewModes.push({
+      id: 'all',
+      label: `All`,
+      subLabel: TreemapUtil.formatBytes(this.currentTreemapRoot.resourceBytes),
+    });
+
+    if (this.currentTreemapRoot.unusedBytes !== undefined) {
+      /** @type {LH.Treemap.NodePath[]} */
+      const highlightNodePaths = [];
+      TreemapUtil.walk(this.currentTreemapRoot, (node, path) => {
+        // Only highlight leaf nodes of a certain size.
+        if (node.children) return;
+        if (!node.unusedBytes || node.unusedBytes < 50 * 1024) return;
```

Comment:
extract this 50KiB to a constant? feels like it will get buried otherwise :)

Reply:
also should this account for relative usage some? like if 90% of a module 500kb dep is used, feels a bit weird to flag it as unused (though a bloat visualization should 😆 )

Reply:
Copied the constants from the unused-audit for this, but I also refactored things a bit...

Now, it will only highlight anything in a d1 (depth one) node if it is greater than some number of unused bytes. The audit uses a 20KiB threshold to ignore an entire bundle, so I did the same here. For a specific leaf node (a module in a bundle), only highlight if more than half of the bytes are unused.

Later PRs will add a pie graph/something similar to show how much of a node is unused.